src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -53,7 +53,7 @@ case class RawlsBillingProject(projectName: RawlsBillingProjectName, users: Set[
 object UserAuth {
 
   def toWorkspaceAccessGroupName(workspaceName: WorkspaceName, accessLevel: WorkspaceAccessLevel) =
-    s"${workspaceName.namespace}/${workspaceName.name} ${accessLevel}".take(64) // group names have a 64 char limit
+    s"${workspaceName.namespace}/${workspaceName.name} ${accessLevel}"
```

Comment:
toWorkspaceAccessGroupName is used in other places so moved the 64 char limit to where it makes more sense